incubator/hnc/pkg/reconcilers/hnc_config.go

```diff
@@ -235,6 +241,25 @@ func (r *ConfigReconciler) syncActiveReconcilers(ctx context.Context, inst *api.
 	return nil
 }
 
+// isTypeCofigured keeps track of whether the configuration of a type already exists.
+func (r *ConfigReconciler) isTypeConfigured(inst *api.HNCConfiguration, t api.TypeSynchronizationSpec,
+	mp map[schema.GroupVersionKind]bool) bool {
+	gvk := schema.FromAPIVersionAndKind(t.APIVersion, t.Kind)
+	if mp[gvk] {
+		specMsg := fmt.Sprintf("APIVersion: %s, Kind: %s, Mode: %s", t.APIVersion, t.Kind, t.Mode)
+		r.Log.Info(fmt.Sprintf("Ignoring the configuration: %s", specMsg))
+		condition := api.HNCConfigurationCondition{
+			Code: api.MultipleConfigurationsForOneType,
+			Msg: fmt.Sprintf("Ignore the configuration: %s because the configuration of the type already exists; "+
+				"only the first configuration will be applied", specMsg),
+		}
+		inst.Status.Conditions = append(inst.Status.Conditions, condition)
```

Comment:
Just confirming, if there are 3 configurations for the same type, the `Status.Conditions` will have 2 same condition for the 2nd and 3rd configurations, right?

Reply:
Yes. The message will show what 2nd and 3rd configurations are.